Class Name Renaming

in https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/intro?view=aspnetcore-3.1&tabs=visual-studio-code#the-sample-app
    at "to run the app after downloading the completed project:"
        from {{Visual Studio Code}} tab
        at "Delete Startup.cs, and rename StartupSQLite.cs to Startup.cs."

didn't you forget to mention to also rename class name in [StartupSQLite.cs]??
I kept getting [ error CS0246: The type or namespace name 'Startup' could not be found (are you missing a using directive or an assembly reference?) ] in the dotnet build command.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7a54c97f-ddd4-b041-5e5f-a2df009ace50
* Version Independent ID: fa488196-1f98-08ea-adcc-17d9dac86727
* Content: [Razor Pages with Entity Framework Core in ASP.NET Core - Tutorial 1 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/intro?view=aspnetcore-3.1&tabs=visual-studio-code)
* Content Source: [aspnetcore/data/ef-rp/intro.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/data/ef-rp/intro.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

--- a/aspnetcore/data/ef-rp/intro.md
+++ b/aspnetcore/data/ef-rp/intro.md
@@ -75,7 +75,7 @@ To run the app after downloading the completed project:
 To run the app after downloading the completed project:
 
 * Delete *ContosoUniversity.csproj*, and rename *ContosoUniversitySQLite.csproj* to *ContosoUniversity.csproj*.
-* Delete *Startup.cs*, and rename *StartupSQLite.cs* to *Startup.cs*.
+* In *Program.cs*, comment out `#define Startup` so `StartupSQLite` is used.
 * Delete *appSettings.json*, and rename *appSettingsSQLite.json* to *appSettings.json*.
 * Delete the *Migrations* folder, and rename *MigrationsSQL* to *Migrations*.
 * Do a global search for `#if SQLiteVersion` and remove `#if SQLiteVersion` and the associated `#endif` statement.

--- a/aspnetcore/data/ef-rp/intro/samples/cu30/Program.cs
+++ b/aspnetcore/data/ef-rp/intro/samples/cu30/Program.cs
@@ -1,3 +1,5 @@
+#define Startup
+
 using ContosoUniversity.Data;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.AspNetCore.Hosting;
@@ -41,7 +43,11 @@ namespace ContosoUniversity
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
+#if Startup
                     webBuilder.UseStartup<Startup>();
+#else
+                    webBuilder.UseStartup<StartupSQLite>();
+#endif
                 });
     }
 }

